ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/individual-journey.ts

```diff
@@ -73,24 +79,8 @@ export class IndividualJourney extends JourneyBase {
 
     let nextStep = this.stepOrder[currentStep + 1];
 
-    // access to a computer.
-    if (this.model.computer === false) {
-      this.submit();
-      nextStep = IndividualJourneySteps.ThankYou;
-    }
-    // access to a camera and microphone.
-    if (this.model.camera === HasAccessToCamera.No) {
-      this.submit();
-      nextStep = IndividualJourneySteps.ThankYou;
-    }
-    // access to the internet.
-    if (this.model.internet === false) {
-      this.submit();
-      nextStep = IndividualJourneySteps.ThankYou;
-    }
-    // consent.
-    if (this.model.consent.answer === true || this.model.consent.answer === false) {
-      this.submit();
+    if (this.submitService.isDropOffPoint(currentStep, this.model)) {
```

Comment:
looks a lot cleaner 👍

Reply:
Check the comment below though, it looks like this code should be something along the lines of:
```typescript
if (this.submitService.isDropOffPoint(currentStep, this.model)) {
  this.submitService.submit(currentStep, this.model);
  this.isSubmitted = true;
  nextStep ...
}
```